hikari/channels.py

```diff
@@ -42,9 +43,10 @@
     "GuildStoreChannel",
     "GuildVoiceChannel",
     "GuildStageChannel",
+    "WebhookChannelT",
```

Comment:
No need to export these

Reply:
There won't be any ref for what it refers to in the docs if it's not exported

Reply:
Then the `__init__.py` would have to be changed to not export these to the root, as we don't put typehints there

Reply:
This was never an issue in the past so I don't see what's changed and if you really want to remove all type variables from the top level the go ahead ig but they're probably more useful to than most of the classes which are exported on the top level so...